runtime/build.gradle

```diff
@@ -48,15 +48,14 @@ kotlin {
     nativeTest{}
   }
 
+  js {
+    useCommonJs()
+    nodejs {}
+    browser {}
+  }
+
   targets {
     targetFromPreset(presets.jvm, 'jvm')
-    targetFromPreset(presets.js, 'js') {
-      tasks.getByName(compilations.main.compileKotlinTaskName).kotlinOptions {
-        moduleKind = 'umd'
```

Comment:
Why was this removed?

Reply:
It was replaced with the `js` block to define the target
Sorry I'm not sure why I did this